recipes/libsemigroups-python-bindings/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "libsemigroups-python-bindings" %}
+{% set version = "0.2.2" %}
+{% set sha256 = "7764bedf64503dcb10679394d8b0df8581b791829e1aebee70bc8cf31f4309f2" %}
+{% set repo = "https://github.com/james-d-mitchell/libsemigroups" %}
+{% set home = "https://james-d-mitchell.github.io/libsemigroups/" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  # The python bindings are not yet merged in master; fetch them from their git
+  # branch instead
+  #git_url: {{ repo }}.git
+  #git_tag: python-bindings
+  url: {{ repo }}/archive/conda-test.tar.gz 
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win]
+  script: cd python-bindings;  python setup.py install --single-version-externally-managed --record record.txt
```

Comment:
Here's a temporary workaround
```yaml
  script:
    - export CPPFLAGS="${CPPFLAGS} -I${PREFIX}/include"
    - cd python-bindings
    - python setup.py install --single-version-externally-managed --record record.txt
```
This should be fixed once https://github.com/conda-forge/toolchain-feedstock/pull/11 is in